### ISSUE TITLE
[Windows/arm][JitStress] Assertion failed 'defList.IsEmpty()' in System.Numerics.Tests.ComplexTests:Conjugate

### ISSUE THREAD

**Issue by echesakovMSFT** [dotnet/coreclr#18362]
Hit this is in PR #18347 during [Windows_NT arm Cross Checked corefx_jitstress1 Build and Test](https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/arm_cross_checked_windows_nt_corefx_jitstress1_tst_prtest/3/consoleText) and [Windows_NT arm Cross Checked corefx_jitstress2 Build and Test](https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/arm_cross_checked_windows_nt_corefx_jitstress2_tst_prtest/3/consoleText)

```
===========================================================================================================

C:\Users\robox\j\workspace\arm_cross_che---953a3c02\_\fx\bin\tests\System.Runtime.Numerics.Tests\netcoreapp-Windows_NT-Release-arm>set COMPlus_JitStress=1 

C:\Users\robox\j\workspace\arm_cross_che---953a3c02\_\fx\bin\tests\System.Runtime.Numerics.Tests\netcoreapp-Windows_NT-Release-arm>call C:\Users\robox\j\workspace\arm_cross_che---953a3c02\_\fx\bin\testhost\netcoreapp-Windows_NT-Release-arm\dotnet.exe xunit.console.netcore.exe System.Runtime.Numerics.Tests.dll  -xml testResults.xml -notrait Benchmark=true -notrait category=nonnetcoreapptests -notrait category=nonwindowstests  -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing 
xUnit.net console test runner (32-bit .NET Core)
Copyright (C) 2014 Outercurve Foundation.

Discovering: System.Runtime.Numerics.Tests
Discovered:  System.Runtime.Numerics.Tests
Starting:    System.Runtime.Numerics.Tests

Assert failure(PID 3720 [0x00000e88], Thread: 9524 [0x2534]): Assertion failed 'defList.IsEmpty()' in 'System.Numerics.Tests.ComplexTests:Conjugate(double,double)' (IL size 57)

    File: d:\j\workspace\arm_cross_che---23cdb046\src\jit\lsrabuild.cpp Line: 2094
    Image: C:\Users\robox\j\workspace\arm_cross_che---953a3c02\_\fx\bin\testhost\netcoreapp-Windows_NT-Release-arm\dotnet.exe

----- end 12:25:03.32 ----- exit code 123456789 ----------------------------------------------------------
COREFX TEST FAILED
Running: C:\Users\robox\j\workspace\arm_cross_che---953a3c02\_\fx\bin\tests\System.Runtime.Performance.Tests\netcoreapp-Windows_NT-Release-arm\RunTests.cmd C:\Users\robox\j\workspace\arm_cross_che---953a3c02\_\fx\bin\testhost\netcoreapp-Windows_NT-Release-arm
Using C:\Users\robox\j\workspace\arm_cross_che---953a3c02\_\fx\bin\testhost\netcoreapp-Windows_NT-Release-arm as the test runtime folder.
Executing in C:\Users\robox\j\workspace\arm_cross_che---953a3c02\_\fx\bin\tests\System.Runtime.Performance.Tests\netcoreapp-Windows_NT-Release-arm\ 
----- start 12:25:03.43 ===============  To repro directly: ===================================================== 
```

```
===========================================================================================================

C:\Users\robox\j\workspace\arm_cross_che---0fb8fec2\_\fx\bin\tests\System.Runtime.Numerics.Tests\netcoreapp-Windows_NT-Release-arm>set COMPlus_JitStress=2 

C:\Users\robox\j\workspace\arm_cross_che---0fb8fec2\_\fx\bin\tests\System.Runtime.Numerics.Tests\netcoreapp-Windows_NT-Release-arm>call C:\Users\robox\j\workspace\arm_cross_che---0fb8fec2\_\fx\bin\testhost\netcoreapp-Windows_NT-Release-arm\dotnet.exe xunit.console.netcore.exe System.Runtime.Numerics.Tests.dll  -xml testResults.xml -notrait Benchmark=true -notrait category=nonnetcoreapptests -notrait category=nonwindowstests  -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing 
xUnit.net console test runner (32-bit .NET Core)
Copyright (C) 2014 Outercurve Foundation.

Discovering: System.Runtime.Numerics.Tests
Discovered:  System.Runtime.Numerics.Tests
Starting:    System.Runtime.Numerics.Tests

Assert failure(PID 10868 [0x00002a74], Thread: 972 [0x03cc]): Assertion failed 'defList.IsEmpty()' in 'System.Numerics.Tests.ComplexTests:Conjugate(double,double)' (IL size 57)

    File: d:\j\workspace\arm_cross_che---f3754642\src\jit\lsrabuild.cpp Line: 2094
    Image: C:\Users\robox\j\workspace\arm_cross_che---0fb8fec2\_\fx\bin\testhost\netcoreapp-Windows_NT-Release-arm\dotnet.exe

----- end 12:30:43.88 ----- exit code 123456789 ----------------------------------------------------------
COREFX TEST FAILED
Running: C:\Users\robox\j\workspace\arm_cross_che---0fb8fec2\_\fx\bin\tests\System.Runtime.Performance.Tests\netcoreapp-Windows_NT-Release-arm\RunTests.cmd C:\Users\robox\j\workspace\arm_cross_che---0fb8fec2\_\fx\bin\testhost\netcoreapp-Windows_NT-Release-arm
Using C:\Users\robox\j\workspace\arm_cross_che---0fb8fec2\_\fx\bin\testhost\netcoreapp-Windows_NT-Release-arm as the test runtime folder.
Executing in C:\Users\robox\j\workspace\arm_cross_che---0fb8fec2\_\fx\bin\tests\System.Runtime.Performance.Tests\netcoreapp-Windows_NT-Release-arm\ 
----- start 12:30:44.01 ===============  To repro directly: ===================================================== 
```
The issue was introduces sometime between now and May 8, 2018
I am going to disable System.Runtime.Numerics.Tests in #18347 
When the issue is resolved please remove corresponding line in tests/arm/corefx_test_exclusions.txt

**Comment by echesakovMSFT** [comment]
@dotnet/jit-contrib

**Comment by CarolEidt** [comment]
Fixed with #18696
